### ISSUE TITLE
Hbase reset info server is not stopped

### ISSUE THREAD

**Issue by artur-tamazian** [sakserv/hadoop-mini-clusters#33]
I'm using hbase local cluster (same code from examples). When I try to stop hbase cluster and then start it again I get an exception saying `java.net.BindException: Port in use: localhost:8085`. So looks like InfoServer started in HbaseRestLocalCluster is not stopped when HbaseRestLocalCluster.stop() is called.

Additional note: HbaseLocalCluster.getHbaseRestLocalCluster() is public even though HbaseRestLocalCluster is package private.

**Comment by sakserv** [comment]
Thanks for reporting this. I have a fix that I'm working through now to validate. I'll push it shortly and make the fix part of the next release.

**Comment by sakserv** [comment]
@artur-tamazian - I'm unable to recreate the exception locally, but I have added code to explicitly stop the InfoServer and I now see the channel being stopped in the tests.

```
2017-04-30 07:09:41 INFO  HbaseLocalCluster:230 - HBASE: Stopping MiniHBaseCluster
2017-04-30 07:09:41 INFO  log:67 - Stopped SelectChannelConnector@0.0.0.0:8085
```

Also, I've made HbaseRestLocalCluster public, I see no reason we shouldn't.

I'll push these changes here in a few minutes. Please feel free to reopen this issue if you still see this after the changes. Thanks again for the report!

**Comment by sakserv** [comment]
PR #34 merged, closing.

**Comment by artur-tamazian** [comment]
Pls see my comment in PR. Thanks for the fix!

**Comment by sakserv** [comment]
@artur-tamazian - thanks for the review, I'll add a null check before next release.

**Comment by artur-tamazian** [comment]
@sakserv is there an estimated release date yet?

**Comment by sakserv** [comment]
@artur-tamazian - I was hoping to get #35 resolved so I could release the HDP 2.6 version at the same time. I've got notes into several folks in an effort to get to working solution on that issue. If I'm unable to make any progress by next week, I'll drop Knox support for now and get the release out. Sorry for the delay.

**Comment by sakserv** [comment]
Really sorry for the delay here. I've finally released 0.1.12 which includes this fix.

**Comment by artur-tamazian** [comment]
@sakserv thanks a lot!
